lib/lighthouse/benefits_documents/service.rb

```diff
@@ -70,6 +70,7 @@ def submit_document(file, file_params, lighthouse_client_id = nil)
       Lighthouse::DocumentUpload.perform_async(@user.icn, document_data.to_serializable_hash)
     rescue CarrierWave::IntegrityError => e
       handle_error(e, lighthouse_client_id, uploader.store_dir)
+      raise e
```

Comment:
Re-raise original error after logging to sentry. @Tonksthebear / Mobile team will address how they want to handle these errors in the near future